core/domain/exp_domain.py

```diff
@@ -2734,6 +2734,58 @@ def _convert_states_v34_dict_to_v35_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v35_dict_to_v36_dict(cls, states_dict):
+        """Converts from version 35 to 36. Version 36 adds a customization arg
+        for the Math interactions that allows creators to specify the letters
+        that would be displayed to the learner.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        greek_symbols = constants.GREEK_SYMBOLS_LOWERCASE + (
+            constants.GREEK_SYMBOLS_UPPERCASE)
+        greek_letters = constants.GREEK_LETTERS
+
+        for state_dict in states_dict.values():
+            if state_dict['interaction']['id'] in (
+                    'AlgebraicExpressionInput', 'MathEquationInput'):
+                variables = set()
+                for group in state_dict['interaction']['answer_groups']:
+                    for rule_spec in group['rule_specs']:
+                        rule_input = rule_spec['inputs']['x']
+
+                        # Removing all greek letters after adding the
+                        # corresponding symbol to the 'variables' set so that
+                        # the letters in them don't get considered individually.
+                        # For eg. if the expression is 'beta + x', we want the
+                        # variables to be ['β', 'x'] instead of
+                        # ['β', 'b', 'e', 't', 'a', 'x'].
+                        for ind, greek_letter in enumerate(greek_letters):
+                            if greek_letter in rule_input:
+                                rule_input = rule_input.replace(
+                                    greek_letter, '')
+                                variables.add(greek_symbols[ind])
+
+                        for character in rule_input:
+                            if character.isalpha():
+                                variables.add(character)
+
+                customization_args = state_dict[
+                    'interaction']['customization_args']
+                customization_args.update({
+                    'customOskLetters': {
```

Comment:
customOSKLetters?

Reply:
That won't work since `customOSKLetters` would get converted to `custom-oskletters-editor` instead of `custom-osk-letters-editor`.